src/grid/LegacyGrid.tsx

```diff
@@ -838,20 +839,19 @@ export interface ChildrenProperties<T> {
   data: T;
 }
 
-export interface VideoGridProps<T> {
+export interface LegacyGridProps<T, R extends HTMLElement> {
   items: TileDescriptor<T>[];
   layout: Layout;
   disableAnimations: boolean;
-  layoutStates: LayoutStatesMap;
-  children: (props: ChildrenProperties<T>) => ReactNode;
+  Tile: ComponentType<TileProps<T, R>>;
```

Comment:
Nitpick, should `Tile` be lower-cased when it's a field name?  This would also avoid confusion with the `Tile` type on line 58.

Reply:
It seems to be a React convention to capitalize names that refer to components, even field names or prop names. Part of the reason is that a camel case name can't be used to refer to a component in JSX, or else React will think you're trying to reference an "intrinsic element" like `div`, `button`, etc.